### DISCUSSION TITLE
Manufacturer template files

### DISCUSSION THREAD

**Discussion by jaminh** [zwave-js/zwave-js#2649] [Q&A]
Are there any existing conventions for manufacturer specific template files? For example should there just be one file per manufacturer or multiple files, and if there should be multiple files how should those be split up? Does it make sense to organize files by device type (i.e a file for light switch parameters and another for thermostat parameters)?

**Comment by blhoward2** [comment]
There should be one template file as there is frequently overlap between device types. If the file grows large you can divide it into sections as it makes sense. (For example, the Aeotec one has a separate section for the WallSwipe settings.)
